recipes/seeq/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "seeq" %}
+{% set version = "0.0.117" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ab83336f21191c9c66026315de2b2889f3f86cfc44e1475df7b62895be9c1530
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

If (and only if) this is made `noarch: python`:
```suggestion
    - python >=3.6
```